sources/packages/backend/apps/api/src/route-controllers/supporting-user/supporting-user.students.controller.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  Controller,
+  Get,
+  NotFoundException,
+  Param,
+  ParseIntPipe,
+} from "@nestjs/common";
+import { ApiNotFoundResponse, ApiTags } from "@nestjs/swagger";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import {
+  AllowAuthorizedParty,
+  RequiresStudentAccount,
+  UserToken,
+} from "../../auth/decorators";
+import {
+  DynamicFormConfigurationService,
+  SupportingUserService,
+} from "../../services";
+import { ClientTypeBaseRoute } from "../../types";
+import { StudentSupportingUserAPIOutDTO } from "./models/supporting-user.dto";
+import { StudentUserToken } from "apps/api/src/auth";
```

Comment:
Even though we have a few occurrences, we try to use `from "../../auth"` instead of imports coming before src.
Can we have it updated?

Reply:
I will fix it.